packages/utils/upgrade/src/modules/project/project.ts

```diff
@@ -21,6 +21,33 @@ import type {
   IProjectBase,
 } from './types';
 
+export const isPluginProject = (project: unknown): project is IPluginProject => {
```

Comment:
What do you think about having a `utils` export within project to separate the class/main business logic from the utilities that lives outside?